docs/semgrep-app/findings.md

```diff
@@ -96,19 +97,11 @@ The Findings page consists of:
 
 **Triaging** means prioritizing a finding based on a policy or criteria set by your team or organization. While severity is a factor in triage, your organization may define additional criteria based on coding standards, business, or product goals.
 
-Semgrep App assists in the triage process through the use of **comments** and **triage states**:
+Semgrep App assists in the status tracking process through the use of **triage** and **bulk triage**, and automatically marks findings as fixed when they are no longer found. It also removes findings when a user disables or a removes a rule, closes a PR without merging it, or stops scanning a particular file.
```

Comment:
`Status tracking` instead of `triage` may seem confusing here. We would need to define - status of what? Of a finding, code, PRs or MRs? I would also not explain here what Semgrep App does as a whole.

```suggestion
Semgrep App assists in the triage process through the use of **triage** and **bulk triage**:
```